metpy/io/upperair.py

```diff
@@ -17,11 +17,14 @@
 from ._tools import UnitLinker
 from .cdm import Dataset
 from ..calc import get_wind_components
+from ..deprecation import deprecated
 from ..package_tools import Exporter
 
 exporter = Exporter(globals())
 
 
+@deprecated('0.8', addendum=' This function is being moved to the Siphon package.',
+            pending=True)
```

Comment:
Why pending? Doesn't pending mean "this will soon be deprecated"? Rather than, this will be deprecated in 0.6 and removed in 0.8?